src/it/resources/nodes.conf

```diff
@@ -0,0 +1,81 @@
+nodes = [{
+  waves {
```

Comment:
Unfortunately, this can not easily be fixed because of how `chainId` is implemented.